src/WordPress/Blueprints/Runtime/NativePHPRuntime.php

```diff
@@ -10,11 +10,13 @@
 class NativePHPRuntime implements RuntimeInterface {
 
 	public Filesystem $fs;
+    protected string $documentRoot;
```

Comment:
Tabs vs spaces again. Would you be able to also set up PHPCBF to auto-format all those files?

Reply:
[see this response](https://github.com/WordPress/blueprints/pull/25/files/98fdb7dc60d43742bb430ca937d96bf3e69682de#r1503291952)